javascriptv3/example_code/cloudwatch-logs/libs/iam-helper.js

```diff
@@ -30,7 +30,7 @@ const createRole = async (roleName, statement) => {
   return Arn;
 };
 
-export const createLambdaRole = async (roleName) => {
+export const createLambdaRole = (roleName) => {
```

Comment:
You may want to make sure code changes haven't altered line numbers that a README depends on.

Reply:
Done